server/Controllers/reserveringen.cs

```diff
@@ -34,35 +36,77 @@ public bool Exists(int id)
     }
 
     [HttpGet("{id}")]
-    public async Task<ActionResult<Reservering>> Get(int id)
+    public async Task<ActionResult<Reservering>> Get([FromHeader(Name = "Authorization")] string token, int id)
     {
         var value = await context.Reservering.FindAsync(id);
         return value == null ? NotFound() : value;
     }
+
```

Comment:
missing the get all which a user owns

Reply:
What do you mean? when you get a user, you get the reserveringen too. i think just query that instead

Reply:
👌